src/db/migrate.rs

```diff
@@ -488,6 +488,20 @@ pub fn migrate(version: Option<Version>, conn: &mut Client) -> CratesfyiResult<(
             "
                 ALTER TABLE queue DROP COLUMN registry;
             "
+        ),
+        migration!(
+            context,
+            21,
+            // description
+            "Add mark for features that are derived from optional dependencies",
+            // upgrade query
+            "
+                ALTER TYPE feature ADD ATTRIBUTE optional_dependency BOOL;
```

Comment:
Where can I learn more about postgres data types? Will `optional_dependency` be added as NULL? It should be.

Reply:
Yes, which is an issue when loading the features page for existing crates:

```
thread '<unnamed>' panicked at 'error retrieving column 0: error deserializing column 0: a Postgres value was `NULL`'
```

Probably the type needs to be changed to `optional_dependency: Option<bool>`.

Reply:
Yes, I have realized that later. IMO it would make sense to make is false as default. WDYT?

Reply:
I think we shouldn't say something we don't know for sure. We should show an optional dependency/feature tag if the info is there, but just show nothing if we don't know.